### ISSUE TITLE
Python ENV shebangs - possibly update references to python to python3

### ISSUE THREAD

**Issue by carlashley** [munki/munki#1120]
With macOS Monterey now more active in notifying users that Python 2 is deprecated, there are three scripts that still reference `/usr/bin/env python` that might need to be updated for Python 3 and a new shebang.

```
[jappleseed@local]:munki # grep -ir "env python" | grep -v python3
./code/server/getmanifest.py:#!/usr/bin/env python
./code/client/repoclean:#!/usr/bin/env python
./code/client/makecatalogs:#!/usr/bin/env python
```

**Comment by gregneagle** [comment]
The original reason for these shebangs was that these tools could be successfully run on Linux to help manage a Munki repo sitting on a Linux server. AFAIK they still can, but the only one I use on Linux is makecatalogs.

It might be acceptable to just change the shebang lines to `#!/usr/local/munki/munki-python` and tell anyone who wants to use the management tools on Linux to manually change the shebang lines to whatever they need to.

**Comment by gmarnin** [comment]
Yes, I use `makecatalogs` on IIS

**Comment by homebysix** [comment]
On macOS 12.3, this looks to be a problem. Updating to the munki-python shebang should resolve it.

```
% makecatalogs --version              
env: python: No such file or directory
% /usr/local/munki/munki-python /usr/local/munki/makecatalogs --version
5.6.1.4393
```

**Comment by homebysix** [comment]
Looks like this was resolved in e80604f2a255bd5cb27cfec3132207a0bbd801b0.
